pavelib/quality.py

```diff
@@ -272,7 +272,12 @@ def run_jshint(options):
     jshint_report = jshint_report_dir / "jshint.report"
     _prepare_report_dir(jshint_report_dir)
 
-    jshint_directories = ["common/static/js", "cms/static/js", "lms/static/js"]
+    jshint_directories = [
```

Comment:
Uck, a hardcoded list of directories. This explains why jshint never seems to catch anything (I didn't think it was even supposed to work in master!).

There is javascript in a lot of other places, and we need to be checking. In particular, the new pattern is for JS to live with it's app. So for example, lms/djangoapps/teams/static has JS.

Why can't we just crawl the general directory structure? What do we for Python pylint/pep8?